Do not fail if the schema validation fails. Ubuntu ignores this by patching
glib to never show the error about deprecated paths.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -178,7 +178,7 @@
 execute_process (COMMAND ${_glib_comple_schemas} --dry-run --schema-file=${CMAKE_CURRENT_SOURCE_DIR}/${UNITY_SCHEMAS} ERROR_VARIABLE _schemas_invalid OUTPUT_STRIP_TRAILING_WHITESPACE)
 
 if (_schemas_invalid)
-  message (SEND_ERROR "Schema validation error: ${_schemas_invalid}")
+  message ("Schema validation error: ${_schemas_invalid}")
 endif (_schemas_invalid)
 
 # Actually install and recomple schemas
